File: calc.php
<?php 
echo "pls enter your first number\n";
$num1 = rtrim(fgets(STDIN));
echo "pls enter your second number\n";
$num2 = rtrim(fgets(STDIN));
echo "pls enter your sign\n";
$sign = rtrim(fgets(STDIN));

if ($sign == "+") {
  $result = (int)$num1 + (int)$num2;
  echo "your answer is $result \n";
}
if ($sign == "-") {
  $result = (int)$num1 - (int)$num2;
  echo "your answer is $result \n";
}
if ($sign == "*") {
  $result = (int)$num1 * (int)$num2;
  echo "your answer is $result \n";
}
if ($sign == "/") {
  $result = (int)$num1 / (int)$num2;
  echo "your answer is $result \n";
}
?>

File: cpp calc/calc.cpp
#include <iostream>

int main()
{
    char sign;
    int num1;
    int num2;

    std::cout << "pls enter your first number\n";
    std::cin >> num1;
    std::cout << "pls enter your second number\n";
    std::cin >> num2;
    std::cout << "pls enter your sign\n";
    std::cin >> sign;

    if (sign == '+')
    {
        float res = num1 + num2;
        std::cout << res;
    }

    else if (sign == '-')
    {
        float res = num1 - num2;
        std::cout << res;
    }

    else if (sign == '*')
    {
        float res = num1 * num2;
        std::cout << res;
    }

    else if (sign == '/')
    {
        float res = num1 / num2;
        std::cout << res;
    }
    return 0;
}
// cpp rating scary😨/10